### ISSUE TITLE
Persistent loop setting

### ISSUE THREAD

**Issue by peteroupc** [pedrolcl/dmidiplayer#21]
The Drumstick player should have a loop button that behaves similarly to other audio players.  When the button is pressed, every MIDI file played from then on will loop until the button is pressed again.  Thus, for example, the loop option persists across MIDI files and even when the Drumstick player is closed and opened again.

- For this loop button, by default, the loop start and loop end should be the beginning and end of the MIDI file.  The loop start and end can instead be those given in the MIDI file if the MIDI file follows certain loop-point conventions such as the so-called RPG Maker convention (see also libsdl-org/SDL_mixer#101).

Currently, in order to loop the playback of a MIDI file, the Loop option must first be chosen from the File menu.  The MIDI file will then loop until the MIDI file is closed or another MIDI file is opened.  In order to loop the new MIDI file, the Loop option must be selected again, which I find cumbersome if I want every MIDI file I play to loop.

**Comment by pedrolcl** [comment]
> Currently, in order to loop the playback of a MIDI file, the Loop option must first be chosen from the File menu.

You can create a loop button in the toolbar, using the "Tool Bar Editor"  (Settings -> Customize Toolbar...)
https://dmidiplayer.sourceforge.io/manual.shtml#toolbar-customization

![image](https://github.com/user-attachments/assets/94213b95-bd9f-4fbb-81ec-3e105de643c8)

There is another related function: File->Playlist repeat, which has 3 options: Nothing, Current Song, and Whole Playlist.

**Comment by peteroupc** [comment]
Indeed, there was a button titled Loop in the bottom of the window all along; the width the window takes on by default apparently hid it out of sight.

And indeed, Playlist Repeat > Current Song already fulfills the persistent loop setting I was seeking.

Thus, most of this issue is resolved for me.

Now all that remains is for the player to support MIDI files that follow certain loop-point conventions such as the so-called RPG Maker convention (control change message 111 marks where to restart the MIDI file when playback reaches the end of the file).  But that is probably out of scope for this issue unless you want to implement those conventions.
